client/src/constants.js

```diff
@@ -1,4 +1,9 @@
+// POSTS_PER_PAGE: how many posts are displayed per one page
 const POSTS_PER_PAGE = 20;
+// REFRESH_TOKEN_TIME: how often a request gets sent to the server to create a new access token.
+// Make sure this number is less the the token expiration time in .env
 const REFRESH_TOKEN_TIME = 600000;
```

Comment:
If you ask to make the check, just get the env variable and adjust this number instead of the comment.